airship.cabal

```diff
@@ -26,6 +26,7 @@ executable airship
                       , http-types >= 0.7
                       , mtl == 2.2.*
                       , text
+                      , time >= 1.5
```

Comment:
do we need >= 1.5? Might prevent some other projects from building that are using 1.4.x.